src/main/java/com/patchworkmc/manifest/accesstransformer/AccessTransformerList.java

```diff
@@ -0,0 +1,61 @@
+package com.patchworkmc.manifest.accesstransformer;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.patchworkmc.manifest.api.Remapper;
+
+public class AccessTransformerList {
+	private List<AccessTransformerEntry> entries;
+
+	public AccessTransformerList(List<AccessTransformerEntry> entries) {
+		this.entries = entries;
+	}
+
+	public static AccessTransformerList parse(Path accessTransformer) {
```

Comment:
We need to check for wildcards/class entries and throw `UnsupportedOperationException`s for now.